cmd/helm-gcs/cmd/push.go

```diff
@@ -43,14 +44,15 @@ var pushCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		return r.PushChart(chartpath, flagForce, flagRetry, flagPublic, flagPublicURL)
+		return r.PushChart(chartpath, flagForce, flagRetry, flagPublic, flagNoIndexCache, flagPublicURL)
 	},
 }
 
 func init() {
 	rootCmd.AddCommand(pushCmd)
 	pushCmd.Flags().BoolVar(&flagForce, "force", false, "upload the chart even if already indexed")
 	pushCmd.Flags().BoolVar(&flagRetry, "retry", false, "retry if the index changed")
+	pushCmd.Flags().BoolVar(&flagNoIndexCache, "noIndexCache", false, "set Cache-Control to no-cache and max-age to 0, avoids index.yaml being cached to speed up when new helm charts can be fetched")
```

Comment:
maybe the default should be `true` as most folks would like their pushed charts to be fetchable straight away?